src/nlinkfs.c

```diff
@@ -446,8 +446,7 @@ static int nlinkfs_open(const char *path, struct fuse_file_info *fi)
  */
 static int nlinkfs_read(const char *path, char *buf, size_t size, off_t offset, struct fuse_file_info *fi)
 {
-	lseek(fi->fh, offset, SEEK_SET);
-	return read(fi->fh, buf, size);
+        return pread(fi->fh, buf, size, offset); 
```

Comment:
Please, fix indentation here, there are extra spaces at the begin and one extra space at the end of this line.